test/requestContextPlugin.spec.js

```diff
@@ -26,9 +26,7 @@ describe('requestContextPlugin', () => {
             function prepareReply() {
               return testService.processRequest(requestId).then(() => {
                 const storedValue = req.requestContext.get('testKey')
-                reply.status(204).send({
-                  storedValue,
-                })
+                reply.status(204).header('storedvalue', storedValue).send()
```

Comment:
The solution LGTM
I preferred the #177 because the `204` status code was not necessary to make the module work, so it was adding confusion and it is better to remove it instead.

If you would like to update this PR adding an clear test such `the plugin must keep the context across 20x requests` it would be best!

Reply:
Hey, so something like the plugin must keep the context across 204 requests => then use the headers in a separate case? @Eomm

Reply:
i have added a new case, does it make sense? what changes would you recommend 🤔 ?